packages/core/theme/src/utilities/clip-path.ts

```diff
@@ -0,0 +1,18 @@
+export default {
+  /**
+   * Clip Path utilities
+   */
```

Comment:
this comment doesn't help much since the file path indicates that already. Either remove it or add some meaningful comments.

Reply:
Hi 
I have updated the comments. Please have a look.